src/main/g8/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-addSbtPlugin("com.mojolly.scalate" % "xsbt-scalate-generator" % "0.5.0")
```

Comment:
@takezoe use this instead: https://oss.sonatype.org/content/repositories/releases/org/scalatra/scalate/sbt-scalate-precompiler_2.10_0.13/1.8.0.1/